examples/wget/wget_main.c

```diff
@@ -152,13 +154,25 @@ int main(int argc, FAR char *argv[])
   ctx.buflen = 512;
   ctx.sink_callback = callback;
   ctx.sink_callback_arg = NULL;
-  if (argc > 1)
+
+  /* mode 0644 */
+
+  mode_t mode = S_IRWXU | S_IRWXG | S_IRWXO;
+  char *filename = argv[2];
+
+  if (argc == 2)
     {
       ctx.url = argv[1];
+      fd = 1;
     }
   else
     {
-      ctx.url = CONFIG_EXAMPLES_WGET_URL;
```

Comment:
The new logic require the user always pass url from the command line. If so, should we remove  EXAMPLES_WGET_URL from Kconfig too?

Reply:
Ok, please update Kconfig in this patch too